lib/trusona/api/verified_response.rb

```diff
@@ -10,12 +10,12 @@ class VerifiedResponse
 
       def initialize(unverified)
         @unverified = unverified
+        @code       = unverified.code
         @verified   = verify
-        @code = unverified.code
       end
 
       def to_h
-        JSON.parse(@unverified.body)
+        JSON.parse(@unverified.body) rescue {}
```

Comment:
Is there a chance that this would swallow and hide an error that we would want to bubble up?